composer.json

```
@@ -17,12 +17,13 @@
     "prefer-stable": true,
     "require": {
         "php": "^7.4|^8.0",
-        "friendsofsymfony/rest-bundle"   : "~1.1||^2.0",
+        "friendsofsymfony/rest-bundle" : "^3.3",
+        "symfony/http-foundation": "^5.4|^6.0",
```

See #19: this bundle uses a few Symfony components, I think they should be declared here
```suggestion
        "friendsofsymfony/rest-bundle" : "^3.3",
        "symfony/config": "^5.4|^6.0",
        "symfony/dependency-injection": "^5.4|^6.0",
        "symfony/http-foundation": "^5.4|^6.0",
```

---

Done thanks.